docs/root/install/tools/route_table_check_tool.rst

```diff
@@ -18,6 +18,10 @@ Input
      :ref:`config <config_tools_router_check_tool>`.
      The tool config input file specifies urls (composed of authorities and paths)
      and expected route parameter values. Additional parameters such as additional headers are optional.
+     Schema: The tool deals with JSON objects internally, based on `this schema <https://github.com/envoyproxy/envoy/blob/master/test/tools/router_check/json/tool_config_schemas.cc>`_.
```

Comment:
Use a `:repo:` link here and elsewhere in this change. Search for other examples.